src/app/pages/task-calendar/rsync/rsync-form/rsync-form.component.ts

```diff
@@ -189,20 +194,21 @@ export class RsyncFormComponent {
         type: 'textarea',
         name: 'rsync_extra',
         placeholder: T('Extra options'),
-        tooltip: T('Add any other <a\
+        tooltip: T('Additional <a\
                     href="https://rsync.samba.org/ftp/rsync/rsync.html"\
-                    target="_blank">rsync(1)</a> options. The "*"\
-                    character must be escaped with a backslash (\\*.txt)\
-                    or used inside single quotes(\'*.txt\').'),
+                    target="_blank">rsync(1)</a> options to include.<br>\
+                    Note: The "*" character <b>must</b> be escaped with\
```

Comment:
The emphasis on *must* seems unnecessary.

Reply:
removed emphasis